src/main/scala/net/snowflake/spark/snowflake/SnowflakeJDBCWrapper.scala

```diff
@@ -177,6 +179,21 @@ private[snowflake] class JDBCWrapper {
     DriverManager.getConnection(jdbcURL, jdbcProperties)
   }
 
+  private def setupJDBCOptions(params: MergedParameters, jdbcProperties: Properties): Unit = {
```

Comment:
Why are only these two options in this method? If we define a method, we should set all of the properties in here.

Reply:
The reason is only these two options are used in this case. It looks that it is not necessary to use a separate function. I will change to set parameter in getConnector().